cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1374,15 +1374,6 @@ def test_advanced_components(self):
         self.assertNotEqual(only_template.get('category'), 'video')
         self.assertNotEqual(only_template.get('category'), 'openassessment')
 
-    def test_advanced_components_without_display_name(self):
```

Comment:
@sarina , I think we want to keep this test, just instead of using `graphical_slider_tool`, use another advanced module? (another good spot to have a generic 'advanced' block)